tests/fixtures/synthetic_hf_state.py

```diff
@@ -0,0 +1,100 @@
+from collections.abc import Iterable
+
+import pytest
+import torch
+from transformers import PreTrainedTokenizer
+
+from composer.core.state import State
+from composer.datasets.dataloader import DataLoaderHparams
+from composer.datasets.lm_datasets import LMDatasetHparams
```

Comment:
nit: I think these can be combined into one line

Reply:
so you want to combine all the calls to `composer.datasets` instead of splitting them out into sub-sub folder import calls?